types/node-horseman/tsconfig.json

```diff
@@ -0,0 +1,23 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6",
+            "dom"
+        ],
+        "noImplicitAny": true,
+        "noImplicitThis": true,
+        "strictNullChecks": true,
+        "baseUrl": "../",
```

Comment:
Also add `"strictFunctionTypes": true,`

Reply:
Hi, thank you so much for helping me out. I've made and pushed the changes as stated. Should I create a new PR? Kindly let me know how to proceed. Thanks a ton!

Reply:
No, making the changes here is good.